src/ScheduledJobs/JobHandlers/ParticipantExportJobHandler.cs

```diff
@@ -33,34 +39,48 @@ public ParticipantExportJobHandler(IS3Service s3Service, ICsvUtilities csvUtilit
             _logger = logger;
         }
 
-        public async Task<bool> HandleAsync(ParticipantExport source)
+        public async Task<bool> HandleAsync(ParticipantExport source, CancellationToken cancellationToken = default)
```

Comment:
This is the one area we need to be careful. Make sure `Dte.Common.Lambda.Resolvers.HandlerResolver` and any other relevant code can pick up this method with the new signature. Looks ok.